internal/extensions/extensions.go

```diff
@@ -29,6 +29,7 @@ type Extensions []string
 // Populate internal extensions here.
 var internalExtensions = Extensions{
 	"internal:runtime_extension_v1",
+	"internal:rename_core_endpoints",
```

Comment:
As we are about to release the LTS, having this API extension is optional right?